cluster/swarm/utils.go

```diff
@@ -33,3 +34,13 @@ func convertMapToKVStrings(values map[string]*string) []string {
 	}
 	return result
 }
+
+var imageEngineOSErrorPattern = regexp.MustCompile(`cannot load (.+) image on (.+)`)
```

Comment:
To the best of my knowledge, that error only occurs during image loads for operating system mismatches, but perhaps the engine folks would know more.